index.html

```diff
@@ -1925,6 +1925,68 @@
         </table>
       </section>
     </section>
+    <section data-dfn-for="Localizable" data-link-for="Localizable">
+      <h2><dfn>Localizable</dfn> dictionary</h2>
+      <pre class="idl">
+        dictionary Localizable {
+          DOMString lang;
+          TextDirection dir = "auto";
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn>lang</dfn> member
+        </dt>
+        <dd>
+          A [[!BCP47]] language tag that specifies the primary language for the
+          values of the human-readable members of the inheriting dictionary.
+        </dd>
+        <dt>
+          <dfn>dir</dfn> member
+        </dt>
+        <dd>
+          Specifies the base direction for the human-readable members of an
+          inheriting dictionary.
+        </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="TextDirection" data-link-for=
+    "TextDirection">
+      <h2>
+        <dfn>TextDirection</dfn> enum
+      </h2>
+      <pre class="idl">
+        enum TextDirection {
+          "auto",
+          "ltr",
+          "rtl"
+        };
+      </pre>
+      <p>
+        The text-direction values are the following, implying that the value of
+        the human-readable members is by default:
+      </p>
+      <dl>
+        <dt>
+          <dfn>auto</dfn>
+        </dt>
+        <dd>
+          Directionality is determined by the [[!BIDI]] algorithm.
```

Comment:
The bidi algorithm should be used for `ltr` and `rtl` as well... right?

I think we basically want to hook into http://www.unicode.org/reports/tr9/#Explicit_Directional_Isolates similarly to what CSS does in https://drafts.csswg.org/css-writing-modes-3/#unicode-bidi (for `isolate` or `plaintext` -- not sure which is more appropriate here).